Tests/SentryTests/Dynamic/Dynamic.swift

```diff
@@ -0,0 +1,315 @@
+//
+//  Dynamic
+//  Created by Mhd Hejazi on 4/15/20.
+//  Copyright © 2020 Samabox. All rights reserved.
+//
+import Foundation
+
+public typealias ObjC = Dynamic
```

Comment:
Every time we have an internal property or method that need to be tested, we must create an extra header to expose those fields to test (eq. SentryHub sampler property). 
With Dynamic we can access it without the extra header.

I copied the code because it needed a small fix to work with or target framework.

Reply:
But I would love to hear your opinion on this.

Reply:
You mean @brustolin we can replace all TestInit headers with that?

Reply:
https://github.com/getsentry/sentry-cocoa/issues/987